docs/GettingStartedDocs/install_oe_sdk-Windows.md

```diff
@@ -2,10 +2,12 @@
 
 ## Platform requirements
 
-- A system with support for SGX1 with Flexible Launch Control (FLC).
-Note: To check if your system has support for SGX1 with or without FLC, please look [here](./SGXSupportLevel.md).
+- A 64 bit machine
 
-- Windows Server 2016 or 2019
+- Check if your system has support for SGX1 with or without FLC, please look [here](./SGXSupportLevel.md)
+    - If your system does not have support for SGX1+FLC, simulation mode an be used.
+
+- Windows Server 2016 or 2019, or Windows 10 (1709 or newer)
```

Comment:
Is Windows 10 supported?

Reply:
It should be. For v0.9.0 @mingweishih validated that he could set up a Windows 10 SGX1 system and use it.